### PR TITLE
8283665: Two Jarsigner tests needs to be updated with JDK-8267319

### PR DESCRIPTION
**Pull request by valeriepeng**
Max, can you please help review this fix? It updates the two jarsigner tests which are added to the main trunk during the code review of JDK-8267319.

Mach5 run succeeds.
Thanks,
Valerie

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8283665](https://bugs.openjdk.java.net/browse/JDK-8283665): Two Jarsigner tests needs to be updated with JDK-8267319

### Reviewers
 * [Xue-Lei Andrew Fan](https://openjdk.java.net/census#xuelei) (@XueleiFan - **Reviewer**)
 * [Anthony Scarpino](https://openjdk.java.net/census#ascarpino) (@ascarpino - **Reviewer**)
 * [Hai-May Chao](https://openjdk.java.net/census#hchao) (@haimaychao - Committer)
 * [Weijun Wang](https://openjdk.java.net/census#weijun) (@wangweij - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7952/head:pull/7952` \
`$ git checkout pull/7952`

Update a local copy of the PR: \
`$ git checkout pull/7952` \
`$ git pull https://git.openjdk.java.net/jdk pull/7952/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7952`

View PR using the GUI difftool: \
`$ git pr show -t 7952`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7952.diff">https://git.openjdk.java.net/jdk/pull/7952.diff</a>

</details>

### CODE REVIEW

**Review by XueleiFan — APPROVED** [review]

**Review by ascarpino — APPROVED** [review]

**Review by haimaychao — APPROVED** [review]

File: test/jdk/sun/security/tools/jarsigner/DisableCurveTest.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
     public static void main(String[] args) throws Exception{
         SecurityTools.keytool("-keystore ks -storepass changeit " +
-                "-genkeypair -keyalg EC -alias ca -dname CN=CA " +
+                "-genkeypair -keyalg EC -keysize 256 -alias ca -dname CN=CA " +
```
Comment (wangweij) [inline test/jdk/sun/security/tools/jarsigner/DisableCurveTest.java:45]: Is this change necessary?

**Review by wangweij — APPROVED** [review]
